packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/request.js

```diff
@@ -1,6 +1,7 @@
 import sinon from 'sinon';
 import {assert} from '@webex/test-helper-chai';
 import MockWebex from '@webex/test-helper-mock-webex';
+
```

Comment:
unnecessary whitespace change, it shouldn't be part of this PR

Reply:
@marcin-bazyl During rebasing I got this extra change and I was not able to remove it due to eslint rules specified.